### PR TITLE
issue-21 TODO for task on testability of gulp-less function

### PR DESCRIPTION
**Pull request by benmcosker**
Confirmed with @paul-barry-kenzan , testing the `gulp-less` configurations for `less-plugin-autoprefix` being passed has not been successful with the existing `lazypipe` module.

cc: @obuckley-kenzan

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -47,7 +62,21 @@ describe('default options', function() {
...
-      .pipe(expect(['dest/pipeline-compile-less.css', 'dest/pipeline-compile-less.css.map']));
+      .pipe(expectFile(['dest/pipeline-compile-less.css', 'dest/pipeline-compile-less.css.map']));
+    done();
+  });
+
+});
+
+describe('autoprefix default options', function() {
+
+  it('Should call the less function', function (done) {
```
Comment (obuckley-kenzan) [inline test/index.spec.js]: we should reference it as `compileLess`

File: test/index.spec.js [hunk 2]
```diff
@@ -47,7 +62,21 @@ describe('default options', function() {
...
     gulp.src(getFixtures('*'))
       .pipe(compilePipeline.compileLESS())
-      .pipe(expect(['dest/pipeline-compile-less.css', 'dest/pipeline-compile-less.css.map']));
+      .pipe(expectFile(['dest/pipeline-compile-less.css', 'dest/pipeline-compile-less.css.map']));
+    done();
+  });
+
+});
+
+describe('autoprefix default options', function() {
```
Comment (obuckley-kenzan) [inline test/index.spec.js]: _autoprefixer_

File: test/index.spec.js [hunk 3]
```diff
@@ -4,7 +4,21 @@ var compilePipeline = require('../src/index.js');
 var clean = require('gulp-clean');
 var gulp = require('gulp');
 var path = require('path');
-var expect = require('gulp-expect-file');
+var expectFile = require('gulp-expect-file');
+
+var less = require('gulp-less');
+var lessSpy;
+
+var sinon = require('sinon');
```
Comment (obuckley-kenzan) [inline test/index.spec.js]: could we just keep all these requires grouped together at the top, alpha sort ASC?

File: test/index.spec.js [hunk 4]
```diff
@@ -47,7 +59,21 @@ describe('default options', function() {
...
+      .pipe(expectFile(['dest/pipeline-compile-less.css', 'dest/pipeline-compile-less.css.map']));
+    done();
+  });
+
+});
+
+describe('autoprefixer default options', function() {
+
+  it('Should call the gulp-less function', function (done) {
+
```
Comment (benmcosker) [inline test/index.spec.js]: It is the gulp-less function being called 
https://github.com/kenzanlabs/pipeline-compile-less/blob/master/src/index.js#L44

Comment (obuckley-kenzan) [inline test/index.spec.js]: Ah, I see.  thanks for clarifying.

**Comment by obuckley-kenzan** [comment]
related to #22 and #23

**Pull request merged**